settings/common.py

```diff
@@ -284,7 +284,7 @@
 CLOUDCV_TEAM_EMAIL = "EvalAI Team <team@cloudcv.org>"
 
 slack_webhook = os.environ.get('SLACK_WEBHOOK')
-if slack_webhook == None:
+if slack_webhook is None:
```

Comment:
@yashdusing 
Instead of this check, you can use
```python
slack_webhook = os.getenv('SLACK_WEBHOOK', 'http://testslackwebhook.com/webhook')
```
Please Ref: https://docs.python.org/3/library/os.html#os.getenv

Reply:
Thanks. Simplifies the logic :) :+1: